charts/astronomer/templates/commander/commander-role.yaml

```diff
@@ -129,4 +129,9 @@ rules:
 - apiGroups: ["policy"]
   resources: ["poddisruptionbudgets"]
   verbs: ["create", "delete", "get"]
+{{- if .Values.global.sccEnabled }}
+- apiGroups: ["security.openshift.io"]
+  resources: ["securitycontextconstraints"]
+  verbs: ["create", "delete"]
+{{- end }}
```

Comment:
We probably need this for it to work, but I imagine some enterprises balking about this.

Is this just needed to support the "sudo" use in the Debian airflow images, or something else too?

Reply:
this one is necessary so that commander have permission to create security context constraints in airflow namespaces

Reply:
this resource https://github.com/astronomer/airflow-chart/blob/master/templates/airflow-scc-anyuid.yaml